playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -141,6 +141,7 @@ EDXAPP_AWS_SES_REGION_ENDPOINT: "email.us-east-1.amazonaws.com"
 EDXAPP_LOG_LEVEL: 'INFO'
 
 EDXAPP_MEMCACHE: [ 'localhost:11211' ]
+EDXAPP_MEMCACHED_OPTIONS: {}
```

Comment:
I would change it to say `EDXAPP_CACHE_OPTIONS` since there is the possibility someone might be using a different cache backend.